src/Tracy/assets/Bar/bar.js

```diff
@@ -357,9 +357,22 @@
 		if (!layer.dataset.id) {
 			return;
 		}
+		var reURLInformation = new RegExp('^(https?:)//(([^:/?#]*)(?::([0-9]+))?)(/[^?#]*)');
+	        var oldOpen = XMLHttpRequest.prototype.open;
+	        XMLHttpRequest.prototype.open = function(){
```

Comment:
And what about to send `setRequestHeader('X-Tracy-Ajax', layer.dataset.id)` directly in open() and leave send() unchanged?